trillian/integration/hammer.go

```diff
@@ -492,7 +480,7 @@ func (s *hammerState) addChain(ctx context.Context) error {
 }
 
 func (s *hammerState) addChainInvalid(ctx context.Context) error {
-	choices := []Choice{EmptyChain, PrecertNotCert, NoChainToRoot, UnparsableCert, SignatureNotCorrect}
+	choices := []Choice{EmptyChain, PrecertNotCert, NoChainToRoot, UnparsableCert}
```

Comment:
We're intentionally dropping SignatureNotCorrect test?

Reply:
Yeah, it's pesky for a later refactoring and I think the same code arm is covered by the case when the intermediate is omitted.